.env.example

```
@@ -147,3 +147,4 @@ RERANKING_CACHE_TTL=1800
 
 # Maximum parallel file ingestion workers
 MAX_PARALLEL_INGESTION=5
+\n# Gemini API Key for Antigravity SDK\nGEMINI_API_KEY=""
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The \n# Gemini API Key for Antigravity SDK\nGEMINI_API_KEY key has incorrect delimiter

(IncorrectDelimiter)